docs/en/agent/tracing-metrics-logging.md

```diff
@@ -22,10 +22,11 @@ If you wish to disable a particular plugin to prevent enhancements related to th
 
 The basic configuration is as follows:
 
-| Name                | Environment Key        | Default Value                                                | Description                                                                                                              |
-|---------------------|------------------------|--------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
-| agent.sampler       | SW_AGENT_SAMPLER       | 1                                                            | Sampling rate of tracing data, which is a floating-point value that must be between 0 and 1.                             |
-| agent.ignore_suffix | SW_AGENT_IGNORE_SUFFIX | .jpg,.jpeg,.js,.css,.png,.bmp,.gif,.ico,.mp3,.mp4,.html,.svg | If the operation name of the first span is included in this set, this segment should be ignored.(multiple split by ","). |
+| Name                    | Environment Key            | Default Value                                                | Description                                                                                                              |
+|-------------------------|----------------------------|--------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| agent.sampler           | SW_AGENT_SAMPLER           | 1                                                            | Sampling rate of tracing data, which is a floating-point value that must be between 0 and 1.                             |
+| agent.ignore_suffix     | SW_AGENT_IGNORE_SUFFIX     | .jpg,.jpeg,.js,.css,.png,.bmp,.gif,.ico,.mp3,.mp4,.html,.svg | If the operation name of the first span is included in this set, this segment should be ignored.(multiple split by ","). |
+| agent.trace_ignore_path | SW_AGENT_TRACE_IGNORE_PATH |                                                              | If the operation name of the first span is matching, this segment should be ignored.(multiple split by ",").             |
```

Comment:
If the entry span(first span) of the segment is `GET:/ABC`, in theory, when I set `agent.ignore_suffix=GET:/ABC`, what will happen?

Reply:
`ignore_suffix` is currently split according to `.` to get the suffix for matching, so `GET:/ABC` will not be filtered.

Reply:
I see. Could you add this to the doc? Because in the Java agent, it is using `endOf` only, so, actually, it could use for complete match filter, although we don't recommend it.

Reply:
Done.